code/modules/power/power.dm

```diff
@@ -266,7 +268,8 @@
 	else
 		power_source = C
 		shock_damage = cell_damage
-	var/drained_hp = M.electrocute_act(shock_damage, source, siemens_coeff) //zzzzzzap!
+	var/drained_hp = M.electrocute_act(min(shock_damage, 150), source, siemens_coeff) //zzzzzzap!
```

Comment:
```suggestion
	var/drained_hp = M.electrocute_act(shock_damage, source, siemens_coeff) //zzzzzzap!
```

Reply:
I still think instakilling runners is lame but sure